src/app/components/MediaLoader/index.styles.ts

```diff
@@ -27,29 +28,45 @@ export default {
 
   landscapeFigure: () => css({ aspectRatio: '16 / 9' }),
   portraitFigure:
-    (isEmbedded = false) =>
-    ({ mq }: Theme) => [
+    (pageType: string, isEmbedded = false) =>
```

Comment:
I am concerned that this is starting to get a bit complicated, now that we have 2 variables which can cause the CSS to change. 

Is there perhaps a way in which we can separate this out into separate functions?